pyproject.toml

```diff
@@ -61,7 +61,7 @@ test_all = [
     "ipykernel",
     "ipython>=7.32",
     "ipywidgets",
-    "coverage[toml]",
+    "coverage[toml]>=7.4.2",
```

Comment:
I guess the real constraint would be `!=7.4.0,!=7.4.1` which are the only two versions that know about the environment variable but don't have a graceful fallback mechanism for it, but this seemed tidier.